ui-modules/blueprint-composer/app/components/designer/designer.directive.js

```diff
@@ -112,22 +114,33 @@ export function designerDirective($log, $state, $q, iconGenerator, catalogApi, b
             switch(toState) {
                 case graphicalEditEntityState:
                     id = toParams.entityId;
+                    $scope.selectedEntityDecorator = null;
+                    break;
+                case graphicalEditSensorsState:
+                    id = toParams.entityId;
+                    $scope.selectedEntityDecorator = 'sensors';
```

Comment:
maybe add comment to explain why selectedEntityDecorator have value only in this case